bin/lottery.sh

```diff
@@ -22,14 +22,14 @@ ask_continue() {
 
 # Check the arguments:
 if [[ $# -lt 1 || $# -gt 2 ]]; then
-  echo "Usage: $(basename $0) ROWS [COLUMNS]"
+  echo "Usage: $(basename "$0") <rows> [<columns>]"
```

Comment:
What's the reason for the angle brackets (chevrons) here?
~~I'm not familiar with that style (and I don't even remember having seen this before tbh - but I likely just forgot about it).~~
Nvm, Git seems to use that style in their man pages :o :D
If I look at `man man`, `man ls`, `man gpg`, this seems to be rather uncommon, right?
Using lowercase here as well seems fine (only the coreutils seem to use uppercase for the synopsis in their man pages).

The rest LGTM. It mixes in some other changes though (the one quoting fix and replacing the old-style backquote form for command substitution). A dedicated commit for this would be nice but it's also fine if you don't want to split the changes up.

Reply:
> What's the reason for the angle brackets (chevrons) here?

I thought that style would be more common, but I'm happy to revert, given it doesn't seem to be that common.

> A dedicated commit for this would be nice but it's also fine if you don't want to split the changes up.

Dedicated commits in one pull request or one per pull request?

Reply:
Dedicated commits in one (this) PR would be fine, thanks.

Reply:
Thanks, done.